frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/GraphTableDialog.java

```diff
@@ -27,16 +27,17 @@
 
   public static GraphTableDialog of(IGitMacheteRepositorySnapshot gitMacheteRepositorySnapshot, String windowTitle,
       @Nullable String emptyTableText, @Nullable Consumer<IGitMacheteRepositorySnapshot> okAction, String okButtonText,
-      boolean cancelButtonVisible) {
+      boolean cancelButtonVisible, boolean hasBranchActionHints) {
     return of(gitMacheteRepositorySnapshot, windowTitle, /* windowWidth */ 800, /* windowHeight */ 500, emptyTableText,
-        okAction, okButtonText, cancelButtonVisible);
+        okAction, okButtonText, cancelButtonVisible, hasBranchActionHints);
   }
 
   public static GraphTableDialog of(IGitMacheteRepositorySnapshot gitMacheteRepositorySnapshot, String windowTitle,
       int windowWidth, int windowHeight, @Nullable String emptyTableText,
-      @Nullable Consumer<IGitMacheteRepositorySnapshot> okAction, String okButtonText, boolean cancelButtonVisible) {
+      @Nullable Consumer<IGitMacheteRepositorySnapshot> okAction, String okButtonText, boolean cancelButtonVisible,
+      boolean hasBranchActionHints) {
     var table = RuntimeBinding.instantiateSoleImplementingClass(ISimpleGraphTableProvider.class)
-        .deriveInstance(gitMacheteRepositorySnapshot);
+        .deriveInstance(gitMacheteRepositorySnapshot, hasBranchActionHints);
```

Comment:
why `hints` here but `tooltips` there (the method's argument)?

Reply:
Yeah, probably left accidentally

Reply:
Yeas, my fault. But I dunno why IntelliJ's `Ctrl` + `Shift` + `R` didn't find them :thinking:

Reply:
`Ctrl+Shift+{F,R}` is flaky in IntelliJ... dunno why, maybe some caching/indexing issues. `git grep` is more reliable ;)